src/sentry/static/sentry/app/components/gridEditable/styles.tsx

```diff
@@ -0,0 +1,290 @@
+import styled from 'react-emotion';
+
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+const GRID_HEADER_HEIGHT = '45px';
+const GRID_EDIT_WIDTH = '35px';
+const GRID_EDIT_WIDTH_DOUBLE = '70px'; // (2 * GRID_EDIT_WIDTH)
+
+/**
+ * Explanation of z-index:
+ *  - Resizer needs to float above <th> cells to be interactive.
+ *  - Editable needs to float above Resizer to hide the right-most Resizer,
+ */
+const Z_INDEX_RESIZER = '1';
+const Z_INDEX_EDITABLE = '10';
+
+type GridEditableProps = {
+  numColumn?: number;
+  isEditable?: boolean;
+  isEditing?: boolean;
+  isPrimary?: boolean;
+};
+
+/**
+ *
+ * Grid is the parent element for the tableResizable component.
+ *
+ * On newer browsers, it will use CSS Grids to implement its layout.
```

Comment:
We don't need to support browsers that don't support grid if that makes the implementation simpler.

Reply:
There's no added difficulty in implementing it with `table`. I'd say it'll make it easier to implement resizable + draggable columns later on.